src/TrackRowInfoVisComparison.js

```diff
@@ -0,0 +1,233 @@
+import React, { useRef, useCallback, useEffect, useState } from "react";
```

Comment:
An optional suggestion:

How about making this track visually similar to `nominal` type tracks and showing user-annotated categories (e.g., "Positive") as text labels? Initially, the bar can be gray as it is with a placeholder text (e.g., "Not selected"), and whenever a user clicks on a bar, the color and the text changes accordingly.

Reply:
This is a good idea, I have added as a separate issue https://github.com/hms-dbmi/cistrome-explorer/issues/303